IdentityCore/src/parameters/MSIDRequestParameters.h

```diff
@@ -60,6 +60,8 @@
 @property (nonatomic) NSString *intuneApplicationIdentifier;
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->